apigen-maintenance/build.gradle

```diff
@@ -5,11 +5,11 @@ buildscript {
 
 version = 'dummy'
 
-apply plugin: 'com.synopsys.integration.simple'
+apply plugin: 'com.blackduck.integration.simple'
 
 dependencies {
     api 'org.apache.bcel:bcel:6.5.0'
 
-    implementation 'com.synopsys.integration:integration-common:24.0.0'
+    implementation 'com.blackduck.integration:integration-common:24.0.0'
```

Comment:
I believe we'll need to specify a new version here.

Reply:
Yes, I would be specifying them in the last commit I make after libraries are released in order including copyright updates.